lib/models/StatsModel.js

```diff
@@ -0,0 +1,22 @@
+const { StatsClient } = require('arsenal').metrics;
+
+/**
+ * @class StatsModel
+ *
+ * @classdesc Extend and overwrite how timestamps are normalized by minutes
+ * rather than by seconds
+ */
+class StatsModel extends StatsClient {
+
+    /**
+    * normalize to the nearest interval
+    * @param {object} d - Date instance
+    * @return {number} timestamp - normalized to the nearest interval
+    */
+    _normalizeTimestamp(d) {
```

Comment:
Why is this a private member? I don't see it used in the class itself so I expect it to be called from outside?

Reply:
It is used in the `StatsClient` class, and since other components are reliant on the class, it seemed best to override this helper function locally for our project rather than creating or extending the functionality of `StatsClient`. Utapi relied on this method to normalize timestamps by seconds, but for backbeat, we want it in minutes.

Also, important to note that expiry and intervals should be set by fixed minutes. So for bloomberg release, it is important for configurations of these variables to be handled properly. Maybe it will have to be by fixed minutes? Not too sure at the moment

Reply:
I'm lacking some context to answer this, but thanks for the explanation for the overriden function.